README.md

```diff
@@ -17,9 +18,16 @@ git clone https://github.com/thayes46/aimi.git
 
 Install all dependent packages:
 ```
+pip3 install --upgrade pip
```

Comment:
Probably do _python3 -m pip install --upgrade pip_

Reply:
updated in commit 501d854